src/records.ts

```diff
@@ -312,6 +312,22 @@ export interface TraitsRecord {
    * when using the City or Insights web service or the Enterprise database.
    */
   readonly isp?: string;
+  /**
+   * The mobile country code (MCC) associated with the IP address and ISP.
+   * See https://en.wikipedia.org/wiki/Mobile_country_code.
+   *
+   * This value is only set when using the City or Insights web service or the
```

Comment:
I guess we could use "City Plus" here too. We do have a story for going through other uses in the docs.